### ISSUE TITLE
Plotting with cartopy ironically breaks with `geomap=False`

### ISSUE THREAD

**Issue by coroa** [PyPSA/PyPSA#75]
@FabianHofmann :)

```
Traceback (most recent call last):
   File "/home/pypsa/pypsa-eur/.snakemake/scripts/tmprhs9nf08.plot_network.py", line 127, in <module>
     ax=ax)
   File "/home/.conda/envs/pypsa-eur/lib/python3.7/site-packages/pypsa/plot.py", line 173, in plot
     if use_cartopy and geomap:
 UnboundLocalError: local variable 'use_cartopy' referenced before assignment
 ```

Would you please rewrite
```
if geomap:
         [ ... ]

if use_cartopy and geomap:
         [ .... ]
```

to:

```python
if geomap:
         [......]

         if use_cartopy:
              [......]
```
And make sure, the code still works with `geomap = False`.

Gracias

**Comment by coroa** [comment]
Reported in https://github.com/PyPSA/pypsa-eur/issues/50

**Comment by FabianHofmann** [comment]
yes, that was unfavourable. Should be fixed now

**Comment by coroa** [comment]
Thanks for the quick fix.

Fixed by https://github.com/PyPSA/PyPSA/commit/8a1dd061f5854c7e97d455dff361e1b6a0029872

@FabianHofmann when you add a commit fixing a reported issue make sure to include `(fixes #75)` in the header of the commit message or `Fixes #75` in the body, then they are linked up automatically.

**Comment by FabianHofmann** [comment]
Alright thanks for the tip!
